src/intrinsics/ecma262/JSON.js

```diff
@@ -505,9 +505,16 @@ export default function(realm: Realm): ObjectValue {
     if (value instanceof AbstractValue || (value instanceof ObjectValue && value.isPartialObject())) {
       // Return abstract result. This enables cloning via JSON.parse(JSON.stringify(...)).
       let clonedValue = InternalJSONClone(realm, value);
-      let result = AbstractValue.createTemporalFromTemplate(realm, JSONStringify, StringValue, [clonedValue], {
-        kind: "JSON.stringify(...)",
-      });
+      let result = AbstractValue.createTemporalFromTemplate(
+        realm,
+        JSONStringify,
+        StringValue,
+        [clonedValue],
+        {
+          kind: "JSON.stringify(...)",
+        },
+        true
```

Comment:
What is this? If it's really a thing, can it instead be moved into the named optional args?